src/library-authoring/component-info/ComponentAdvancedInfo.tsx

```diff
@@ -48,72 +48,79 @@ export const ComponentAdvancedInfo: React.FC<Record<never, never>> = () => {
     });
   }, [editorRef, olxUpdater, intl]);
 
+  return (
```

Comment:
Note: I didn't make any changes to any of this code here - just moved it from below.